hummingbot/client/ui/__init__.py

```diff
@@ -101,8 +102,13 @@ def login_prompt():
             return False
         if password != re_password:
             err_msg = "Passwords entered do not match, please try again."
+        elif len(password) == 0:
+            err_msg = "Password not entered. Please enter password."
```

Comment:
We don't need to check for empty password, it was discussed before. Basically we don't check for any password strength at all.

Reply:
Done!